src/Microsoft.AspNetCore.Http.Extensions/UriHelper.cs

```diff
@@ -195,16 +195,16 @@ public static string GetEncodedPathAndQuery(this HttpRequest request)
         /// <returns></returns>
         public static string GetDisplayUrl(this HttpRequest request)
         {
-            var host = request.Host.Value;
-            var pathBase = request.PathBase.Value;
-            var path = request.Path.Value;
-            var queryString = request.QueryString.Value;
+            var host = request.Host.Value ?? "";
+            var pathBase = request.PathBase.Value ?? "";
+            var path = request.Path.Value ?? "";
+            var queryString = request.QueryString.Value ?? "";
 
             // PERF: Calculate string length to allocate correct buffer size for StringBuilder.
             var length = request.Scheme.Length + SchemeDelimiter.Length + host.Length
                 + pathBase.Length + path.Length + queryString.Length;
 
-            return new StringBuilder(length)
+            return new StringBuilder()
```

Comment:
Revert, that's the whole point in pre-calculating the length.

Reply:
oops you are right, I edited on GH directly and copy pasted the wrong thing 🤠